canonical-web-and-design/maas-ui
# Convert KVM React code to TypeScript

## Done
- Converted all files under `app/kvm/` to TypeScript.
- Converted reducer and selector files that are relevant to KVM page to TypeScript (pod, resourcepool, zone).
- Added `app/base/types.ts` file which currently holds all the types used in more one place.

## QA
- See if you like the initial structure of the code.
- Check that the app runs.

Fixes #1263
